### DISCUSSION TITLE
Drop connections from banned countries

### DISCUSSION THREAD

**Discussion by TomOdellSheetMusic** [tobychui/zoraxy#424] [Q&A] [answered]
Hello,

First of all: Thank you for your work tobychui!

Is there a way I can drop traffic from the banned countries instead of rejecting it / blocking access?
Im thinking about using the installed nftables or installing iptables for which I have already created such a configuration, but do not want to undermine the architecture without knowing the implications of doing so (Im on a proxmox LXC btw).
Maybe this would be more suitable as a feature request.

**Comment by tobychui** [comment, marked as answer]
@TomOdellSheetMusic Zoraxy works on top of http. When the proxy resolved the country code for the request, it has already accepted the connection. Replying a few (kilo) bytes for showing the access denied page isn't that much different in load compare to directly closing the request.

You should use tools that handle lower network layer like iptable you mentioned to achieve what you need is closing TCP connection directly.
